shenyu-admin/src/main/java/org/apache/shenyu/admin/service/RuleService.java

```diff
@@ -48,6 +52,16 @@ public interface RuleService extends PageService<RuleQueryCondition, RuleVO> {
      * @return rows int
      */
     default int createOrUpdate(final RuleDTO ruleDTO) {
+        // check rule uri condition
+        try {
```

Comment:
> 

Good Idea. I'll try.

Reply:
Hi, I took a look, because RuleConditionDTO is not just a URL rule, there will be other things, so it may not be easy to add validation on a field, do you have a good idea?